custom/icds_reports/management/commands/citus_migration.py

```diff
@@ -0,0 +1,254 @@
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+import inspect
+import logging
+from collections import defaultdict
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from sqlalchemy import inspect as sqlinspect
+
+from corehq.apps.userreports.models import StaticDataSourceConfiguration
+from corehq.apps.userreports.util import get_indicator_adapter, UCR_TABLE_PREFIX
+from corehq.sql_db.connections import connection_manager
+from custom.icds_reports.const import AGG_LS_AWC_VISIT_TABLE, AGG_LS_VHND_TABLE, AGG_LS_BENEFICIARY_TABLE, \
+    AGG_GROWTH_MONITORING_TABLE, AGG_DAILY_FEEDING_TABLE, AGG_COMP_FEEDING_TABLE, AGG_CCS_RECORD_CF_TABLE, \
+    AGG_CHILD_HEALTH_THR_TABLE, AGG_CCS_RECORD_THR_TABLE, AGG_CHILD_HEALTH_PNC_TABLE, AGG_CCS_RECORD_PNC_TABLE, \
+    AGG_CCS_RECORD_DELIVERY_TABLE, AGG_CCS_RECORD_BP_TABLE, AGG_INFRASTRUCTURE_TABLE, AWW_INCENTIVE_TABLE
+from custom.icds_reports.models import AggregateInactiveAWW, AggregateSQLProfile, ICDSAuditEntryRecord
+from custom.icds_reports.models.helper import IcdsFile
+
+logger = logging.getLogger(__name__)
+
+IGNORE_TABLES = {
+    'django_migrations',
+    AggregateSQLProfile._meta.db_table,
+    'ucr_table_name_mapping',
+}
+
+"""
+- table get's created with normal django migrations
+- create child tables
+- dump and load data from individual child tables
+"""
+keep_child_tables = [
+    'agg_awc',
```

Comment:
i don think this or `agg_awc_daily` below get created with normal migrations, not sure if that matters for this though

Reply:
They get created in `setup.sql` which is the first migration run: https://github.com/dimagi/commcare-hq/blob/c0711d50c559f9c62cce2e4bf6de5ebaab47328e/custom/icds_reports/migrations/0001_squashed_0110_aww_incentive_report_monthly.py#L32